storefront/spec/features/account_spec.rb

```
@@ -0,0 +1,269 @@
+
+      user.reload
+
+      expect(user.first_name).to eq(first_name)
+      expect(user.last_name).to eq(last_name)
+      expect(user.phone).to eq(phone)
+    end
+  end
+
+  describe 'recent orders' do
+    let(:user) { create(:user_with_addresses) }
+    let(:order1) { create(:shipped_order, user: user) }
```

RuboCop (1.75.5)

[convention] This `let` statement uses `1` in its name. Please give it a meaningful name.

(RSpec/IndexedLet)